src/styles/block/card.scss

```diff
@@ -6,13 +6,23 @@
   border-radius: 5px;
 
   &__img {
-    margin: 29px 19px 0;
+    margin: 29px 19px 45px;
     width: 160px;
     height: 134px;
+    transition-duration: 0.4s;
+    transition-property: margin, padding, width, height;
+  }
+
+  &__img:hover {
+    margin: 0 0 37px;
```

Comment:
don't animate width, height, margins and paddings. Use transform scale for changing size and change border-color.
https://pqina.nl/blog/animating-width-and-height-without-the-squish-effect/